File: VB/Custom Wizard/Misc/CustomCloseWizardEventArgs.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.DataAccess.Sql

Namespace CustomDataSourceWizard
	Public Class CustomClosingEventArgs
		Inherits EventArgs

		Public Sub New()
		End Sub
		Public Sub New(ByVal closeMode As CloseMode)
			Me.CloseMode = closeMode
		End Sub
		Public Sub New(ByVal ds As SqlDataSource, ByVal dataMember As String, ByVal closeMode As CloseMode)
			Me.DataSource = ds
			Me.DataMember = dataMember
			Me.CloseMode = closeMode
		End Sub
		Public Property DataSource() As SqlDataSource
		Public Property DataMember() As String
		Public Property CloseMode() As CloseMode
	End Class

End Namespace

File: CS/Custom Wizard/Misc/CustomCloseWizardEventArgs.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.DataAccess.Sql;

namespace CustomDataSourceWizard {
    public class CustomClosingEventArgs : EventArgs{
        public CustomClosingEventArgs() {
        }
        public CustomClosingEventArgs(CloseMode closeMode) {
            this.CloseMode = closeMode;
        }
        public CustomClosingEventArgs(SqlDataSource ds, string dataMember, CloseMode closeMode) {
            this.DataSource = ds;
            this.DataMember = dataMember;
            this.CloseMode = closeMode;
        }
        public SqlDataSource DataSource {
            get;
            set;
        }
        public string DataMember {
            get;
            set;
        }
        public CloseMode CloseMode {
            get;
            set;
        }
    }
    
}
